src/common/definitions.js

```diff
@@ -152,11 +166,16 @@ const pciuAddress = (() => {
         maxLength: 35,
         pattern: "([a-zA-Z0-9\-'.#]([a-zA-Z0-9\-'.# ])?)+$"
       },
+      state: {
+        type: 'string',
+        'enum': usaStates,
+        enumNames: usaStatesLabels
+      },
       zipFirstFive: {
```

Comment:
The UI for this is one field, are we planning to split ZIP up on the FE before submission?

Reply:
Swagger for the submit endpoint lists these as separate fields (both optional). I hadn't really put enough thought into this, but I figure we'll probably want to do whatever `vets-api` will support. My preference would be to submit as one field since that's what's closest to what we need in the front end (i.e., I think we'd only want users to fill out one zip field that accepts the right pattern for all 5 to 9 digits and a dash). I'll reach out to them and get confirmation of what their plan is.

Reply:
Thanks for clarifying, very helpful @markgreenburg .